docs/resources/repo.md

```diff
@@ -3,13 +3,13 @@ subcategory: "Workspace"
 ---
 # databricks_repo Resource
 
-This resource allows you to manage [Databricks Repos](https://docs.databricks.com/repos.html).
+This resource allows you to manage [Databricks Git folders](https://docs.databricks.com/en/repos/index.html) (formerly known as Databricks Repos).
 
--> **Note** To create a Repo from a private repository you need to configure Git token as described in the [documentation](https://docs.databricks.com/repos.html#configure-your-git-integration-with-databricks).  To set this token you can use [databricks_git_credential](git_credential.md) resource.
+-> **Note** To create a Git folder from a private repository you need to configure Git token as described in the [documentation](https://docs.databricks.com/en/repos/index.html#configure-your-git-integration-with-databricks).  To set this token you can use [databricks_git_credential](git_credential.md) resource.
 
 ## Example Usage
 
-You can declare Terraform-managed Repo by specifying `url` attribute of Git repository. In addition to that you may need to specify `git_provider` attribute if Git provider doesn't belong to cloud Git providers (Github, GitLab, ...).  If `path` attribute isn't provided, then repo will be created in the user's repo directory (`/Repos/<username>/...`):
+You can declare Terraform-managed Git folder by specifying `url` attribute of Git repository. In addition to that you may need to specify `git_provider` attribute if Git provider doesn't belong to cloud Git providers (Github, GitLab, ...).  If `path` attribute isn't provided, then Git folder will be created in the user's directory (for existing workspaces it's `/Repos/<username>/...`, for new workspaces it will be user's folder in the workspace):
```

Comment:
@damiangajd-db Has the Repos Create API default location been changed? If not, we should time the merge of this Pr with that change.

Reply:
We haven't changed it yet.

Reply:
@dbbnicole @damiangajd-db I've updated description - please approve if it's ok